pallene/ir.lua

```diff
@@ -165,11 +165,11 @@ local ir_cmd_constructors = {
 
     -- Builtin operations
     BuiltinIoWrite    = {"loc",        "src"},
-    BuiltinMathSqrt   = {"loc", "dst", "src"},
-    BuiltinStringChar = {"loc", "dst", "src"},
-    BuiltinStringSub  = {"loc", "dst", "src1", "src2", "src3"},
-    BuiltinToFloat    = {"loc", "dst", "src"},
-    BuiltinType       = {"loc", "dst", "src"},
+    BuiltinMathSqrt   = {"loc", "dsts", "src"},
+    BuiltinStringChar = {"loc", "dsts", "src"},
+    BuiltinStringSub  = {"loc", "dsts", "src1", "src2", "src3"},
+    BuiltinToFloat    = {"loc", "dsts", "src"},
+    BuiltinType       = {"loc", "dsts", "src"},
```

Comment:
If we are going to change the dsts we might as well change the src1 .. srcN to srcs as well. (And while you are at it, also remove the references to those field named from the `get_srcs` function).